docs/ide/editorconfig-code-style-settings-reference.md

```diff
@@ -1454,6 +1456,34 @@ Example *.editorconfig* file:
 dotnet_sort_system_directives_first = true
 ```
 
+**dotnet\_separate\_import\_directive\_groups**
+
+- When this rule is set to **true**, place a blank line between using directive groups.
+- When this rule is set to **false**, do not place a blank line between using directive groups.
+
+Code examples:
+
+```csharp
+// dotnet_separate_import_directive_groups = true
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+using Octokit;
+
+// dotnet_separate_import_directive_groups = false
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Octokit;
+```
+
+Example *.editorconfig* file:
+
+```EditorConfig
```

Comment:
Should be `ini`

Reply:
Linguist doesn't have `EditorConfig` indeed, but I don't see `ini` being used anywhere in this repo (it's `EditorConfig` everywhere), and that probably would also break the displaying of this on `docs.microsoft.com`, since it's now displayed like this:

![screenshot 2019-02-04 at 21 37 42](https://user-images.githubusercontent.com/460904/52235701-3129c480-28c5-11e9-8914-13c065a496a1.png)

Maybe, @gewarren can give a 2nd opinion.

Reply:
Good point, I didn't noticed it was being used here already. Swapping it to INI should probably done as a separate cleanpu

Reply:
@nschonni Can you maybe make a separate issue for this?